app/builders/digest_email_builder.rb

```diff
@@ -57,7 +57,12 @@ def presented_results
 
   def title_and_optional_url
     result = "# " + digest_item.subscriber_list_title
-    source_url = SourceUrlPresenter.call(digest_item.subscriber_list_url)
+
+    source_url = SourceUrlPresenter.call(
+      digest_item.subscriber_list_url,
+      utm_source: digest_item.subscriber_list_slug,
+      utm_content: digest_run.range,
```

Comment:
Not sure I follow the commit message about the relevance of the utm_content param. Could you explain?

Reply:
Sure, the whole [UTM thing](https://en.wikipedia.org/wiki/UTM_parameters) is quite bizarre, as I think it was designed for links on web pages, rather than links in emails. The overall goal is that we want to know which links are being used in the different emails we send. All the info has to be in the link itself, since that's all we get after the user clicks on it.

In this case, we know that the link came from a particular list (UTM Source). The UTM Content param is optional, and another way to distinguish the link. Here, it will tell us whether the link was in a weekly, daily, or individual-type email. We do a similar thing already for one of the other links ([here](https://github.com/alphagov/email-alert-api/blob/f1687ad93e261a1ec4368072407b5221d68fcb7a/app/presenters/content_change_presenter.rb#L30)).

It's pretty speculative, but it would allow us to see if e.g. users interact with digest emails more than individual-type emails.

In case it helps, I wrote a bit about tracking with UTM params [here](https://github.com/alphagov/content-publisher/blob/master/docs/approach-to-analytics.md#utm--inbound-links).

Reply:
Thanks for the explanation!